src/Microsoft.AspNetCore.SystemWebAdapters/Adapters/Features/IHttpResponseBufferingFeature.cs

```diff
@@ -16,13 +16,15 @@ namespace Microsoft.AspNetCore.SystemWebAdapters.Features;
 [Experimental(Constants.ExperimentalFeatures.DiagnosticId)]
 public interface IHttpResponseBufferingFeature
 {
-    void EnableBuffering(int memoryThreshold, long? bufferLimit);
+    void EnableBuffering(int? memoryThreshold = default, long? bufferLimit = default);
```

Comment:
Do we really need the defaults and nullable changes here? Trying to see if we can avoid being binary breaking here.

Reply:
it makes it easier to just enable buffering when we want the default and the default value is currently in a different layer than I wanted to call it from.

This interface is intentionally marked as "Experimental" so we can break it as much as we need